pym/bob/scm/url.py

```diff
@@ -203,10 +203,13 @@ def getProperties(self):
         return ret
 
     def _download(self, destination):
+        from .. import BOB_INPUT_HASH
+
         # restore signals to default so that Ctrl+C kills us
         signal.signal(signal.SIGINT, signal.SIG_DFL)
 
         headers = {}
+        headers["User-Agent"] = "BobBuildTool (version {})".format(BOB_INPUT_HASH)
```

Comment:
Shouldn't we use the real version here?

```suggestion
        headers["User-Agent"] = "BobBuildTool (version {})".format(BOB_VERSION)
```

Reply:
Yes, probably. 😂 TBH, I didn't really grasp what you were doing in that `__init__.py` file...